build-system/compile/post-closure-babel.js

```diff
@@ -64,7 +64,7 @@ async function postClosureBabel(file) {
   const {compressed, terserMap} = await terserMinify(code);
   await fs.outputFile(file, compressed);
 
-  const closureMap = await fs.readJson(`${file}.map`, 'utf-8');
+  const closureMap = await fs.readJson(`${file}.map`, {encoding: 'utf8'});
```

Comment:
Looks like we're passing in the full options object `{encoding: 'utf8'}` for some operations and the `utf8` shorthand for others. Can we standardize to one or the other for consistency and maintainability?

Reply:
I'd love to standardize on the string version. Unfortunately specifically with `fs-extra` the type annotation is wrong. I'll send a PR to that repo to fix this, once the annotation is corrected I can follow up with a fix

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55845